.requirements

```diff
@@ -7,6 +7,7 @@ RESTY_LUAROCKS_VERSION=3.8.0
 RESTY_OPENSSL_VERSION=1.1.1n
 RESTY_PCRE_VERSION=8.45
 RESTY_LMDB_VERSION=master
+RESTY_EVENTS_VERSION=feat/rebased_new_apis
```

Comment:
We need a proper version before `3.0.0` Kong release (same goes for LMDB) (I know it is because there are other PRs waiting to be merged, but I left this just as an reminder here).

Reply:
Thanks. If events library has main feature ready, I will create a version tag and update here.